fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DataServlet.java

```diff
@@ -404,6 +405,7 @@ public void getData(
                     apiRequest.getAsyncAfter(),
                     apiRequest.getFormat(),
                     uriInfo,
+                    apiRequest,
```

Comment:
Since we have to pass the entire apiRequest into this method anyway, we might as well drop the parameters that we pull directly from the apiRequest (i.e. asyncAfter, format, the uriInfo). It'll make this method's signature a little bit smaller, which is always nice when you can get away with it.